core/storage/email/gae_models.py

```diff
@@ -327,6 +333,11 @@ def get_deletion_policy():
         """Sent email should be kept for audit purposes."""
         return base_models.DELETION_POLICY.KEEP
 
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
```

Comment:
They don't? I see recipient_ids, sender_ids, etc.

Reply:
the ids that are contained here are just the ids of the users to which an email should be sent. I don't think this is relevant user data.

Reply:
It's still user data, just not exported, right?

Reply:
I changed the docstring, but let me know if it is not verbose enough.